examples/cookbook/transactions/get-tx-detail.js

```diff
@@ -16,7 +16,7 @@ const keyStore = new keyStores.UnencryptedFileSystemKeyStore(credentialsPath);
 const config = {
   keyStore,
   networkId: "testnet",
-  nodeUrl: "https://rpc.testnet.near.org",
+  nodeUrl: "https://archival-rpc.testnet.near.org",
```

Comment:
Usually, people do not want to use `archival` nodes. Can we link a note or link to documentation that will describe the difference?